packages/colorpicker/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: colorpicker
+description: "A new Flutter package project."
+version: 0.0.1
+publish_to: "none"
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=1.17.0"
+
+# this package should be built separately and not depend on other app modules
```

Comment:
yes this was requested by @juliajulie95 ma'am
> This package should be completely seperated from the other ones, because it will built seperately too and be integrated in another app which does not have the component library.

Reply:
good.